spec/attributes/README.md

```diff
@@ -0,0 +1,233 @@
+
+#### @can-overlap
+
+_Value:_ `yes` or `no`.
+
+Indicates whether or not the _markup_ and its contents where this _attribute_ is used
+can enclose partial _markup_
+(i.e. a _markup-open_ without its corresponding _markup-end_,
+or a _markup-end_ without its corresponding _markup-start_).
+
+#### #can-reorder
+
+_Value:_ `yes` or `no`.
+
+Indicates whether or not the _markup_ and its contents can be re-ordered.
+
+#### @comment
+
+_Value_: A non-empty string.
+
+Associates a freeform comment with the _markup_.
+
+> For example:
+>
+> ```
+> Click {#link @comment=|Rendered as a button|}here{/link} to continue.
+> ```
+
+#### @term
+
+_Value_: A non-empty string, or a URI.
+
+Identifies a well-defined term.
+The value may be a short definition of the term,
+or a URI pointing to such a definition.
+
+> For example:
+>
+> ```
+> He saw his {#span @term=|https://en.wikipedia.org/wiki/Doppelg%C3%A4nger|}doppelgänger{/span}.
+> ```
+
+#### @translate
+
+_Value:_ `yes` or `no`.
+
+Indicates whether the _markup_ and its contents are translatable or not.
+
+> For example:
+>
+> ```
+> He saw his {#span @translate=no}doppelgänger{/span}.
+> ```
+
+### Message Attributes
+
+#### @allow-empty
+
+_Value:_ `yes` or `no`.
+
+Explicitly mark a message with an empty _pattern_ as valid.
+
+Most empty messages are mistakes,
+so being able to mark ones that can be empty is useful.
+
+Empty _messages_ SHOULD be accompanied by an explanatory `@comment`.
+
+#### @max-length
+
+_Value:_ A strictly positive integer, followed by a space, followed by one of the following:
+- `chars`
+- `lines`
+
+Limits the length of a _message_.
+
+#### @obsolete
+
+_Value:_ `yes` or `no`.
+
+Explicitly mark a _message_ as obsolete.
+
+This might be used in workflows where messages are not immediately removed
+when they are no longer referenced by code,
+but kept in to support patch releases for previous versions.
+During translation, this can be used to de-prioritize such messages.
+
+> [!NOTE]
+> The value could include a way to note some version or timestamp when the removal happened,
+> or be paired with a second `@removed-in` or similar tag.
+
+#### @param
+
+_Value_: **TBD**
+
+Documents a _variable_.
+
+> [!NOTE]
+> Having a well-defined structure for this attribute is pretty important,
+> at least to identify the variable its description is pertaining to.
+> In addition to describing the variable in words, it could include:
+> - The variable's type -- is it a string, a number, something else?
+> - A default example value to use for the variable.
+
+#### @schema
+
+_Value:_ A valid URI.
+
+Identify the _functions_ and _markup_ supported by the _message_ formatter.
+
+#### @source
+
+_Value_: A string.
+
+Provides the _message_ in its source locale.
+
+#### @translate
+
+_Value:_ `yes` or `no`
+
+Indicates whether the _message_ is translatable or not.
+
+Some _messages_ may be required to have the same value in all locales.
```

Comment:
Then they are not messages that should be stored in resource bundles. They can very well be hard-coded.

A better use case is probably to encode info about locale sensitive behavior. For example the fact that the default order for a Contacts app should be first-name, except that Japanese, and a few others should be last name.

But that would not be MF2.

TLDR: I am not sure I see a good use case.